src/components/Image/DetailV2/ImageContestCollectionDetails.tsx

```diff
@@ -49,6 +57,23 @@ export const ImageContestCollectionDetails = ({
               category
             </>
           ) : null;
+          const inReview = item.status === CollectionItemStatus.REVIEW;
+
+          if (isModerator && inReview) {
```

Comment:
This is fine for the time being, but ideally, we should check collection access 😎 will allow anyone with permission to do it.